README.md

```diff
@@ -1,28 +1,70 @@
 # friendbook
 الكود الخاص بمشروع كتاب الأصدقاء 
 
-## مزيد من التفاصيل مع الشرح و التطبيق هنا:
-
-[![IMAGE ALT TEXT HERE](https://img.youtube.com/vi/PrPqBi_lSUc/0.jpg)](https://www.youtube.com/watch?v=PrPqBi_lSUc)ا
 ## طريقة الاستخدام
 
-1. انسخ هذا المشروع clone على جهازك الشخصي فى أى مكان
-2. فى شاشة الأوامر terminal /  command line interface  افتح المجلد و قم بكتابة الأمر   `yarn`  لتثبيت المكتبات التي يحتاجها المشروع للتشغيل
-3. اكتب الأمر `yarn build:release` لبناء و استخراج ملف wasm من الكود استعداداً لرفعه. ستجد بعد تنفيذ هذا الأمر ملف `friendbook.wasm` تم انشاءه فى هذا المسار `build\release` 
-4. قم بكتابة الأمر `near dev-deploy .\build\release\friendbook.wasm` لرفع الملف السابق على البلوك تشين 
```

Comment:
I tried copy pasting this at first and it did not work. @mhassanist is this how slashes are in windows?